packages/js/src/integrations-page/simple-integration.js

```diff
@@ -19,11 +21,15 @@ import { getIsFreeIntegrationOrPremiumAvailable } from "./helper";
 export const SimpleIntegration = ( { integration, isActive, children } ) => {
 	const IntegrationLogo = integration.logo;
 
+	const linkParams = useSelect( select => select( "yoast-seo/settings" ).selectLinkParams(), [] );
+	const learnMoreLink = addQueryArgs( integration.learnMoreLink, linkParams );
+	const logoLink = addQueryArgs( integration.logoLink, linkParams );
```

Comment:
* The `selectLink` selector could do the query params adding for you. With the benefit of caching
* This store actually has a condition (and probably more in PHP) before it is created. Are we sure that we can rely on that?

Reply:
This store is loaded for admin pages